convert2rhel/systeminfo.py

```diff
@@ -354,15 +375,14 @@ def _get_releasever(self):
         releasever_cfg = self._get_cfg_opt("releasever")
         try:
             # return config value or corresponding releasever from the RELEASE_VER_MAPPING
-            return releasever_cfg or RELEASE_VER_MAPPING[".".join(map(str, self.version))]
+            return releasever_cfg or RELEASE_VER_MAPPING[repr(self.version)]
         except KeyError:
             self.logger.critical(
-                "%s of version %d.%d is not allowed for conversion.\n"
+                "%s of version %s is not allowed for conversion.\n"
                 "Allowed versions are: %s"
                 % (
                     self.name,
-                    self.version.major,
-                    self.version.minor,
+                    repr(self.version),
```

Comment:
Here I'd also change to `.format()`
```suggestion
                    self.version,
```